zhaquirks/tuya/tuya_valve.py

```diff
@@ -577,7 +577,7 @@ class GiexIrrigationStatus(t.enum8):
         fallback_name="Total flow reset switch",
     )
     .tuya_sensor(
-        dp_id=102,
+        dp_id=38,
         attribute_name="irrigation_duration",
         type=t.uint32_t,
```

Comment:
dp 38 seems to be a `CharacterString`(?) (similar to how we implement dp 106).
https://github.com/Koenkk/zigbee-herdsman-converters/blob/b8008c9fca769668e7da82ace1dabe602dd40b9b/src/devices/neo.ts#L190

prairiesnpr can hopefully take a second look in a few hours.

Reply:
Fixed by https://github.com/zigpy/zha-device-handlers/pull/3833